### PR TITLE
Fix issue905

### PR DESCRIPTION
**Pull request by bharath-ravindranath**

### CODE REVIEW

**Comment by ktbyers** [comment]
@bharath-ravindranath It looks like you need to rebase your Pull Request.

**Comment by mirceaulinic** [comment]
Hey @bharath-ravindranath - were you willing to revisit this PR? I tried to rebase your PR, but it looks like there are some tests failing as well, and I figured I'll leave this to you to sort it out.

**Comment by mirceaulinic** [comment]
Bump @bharath-ravindranath

**Comment by bharath-ravindranath** [comment]
Not sure what I am doing wrong here. I see a lint failure. But I don't see what the errors are. can someone guide me where to look?

**Comment by itdependsnetworks** [comment]
I would guess there was an issue with travis, would try and make a small change (add a comment or similar) and see if it runs travis again.

**Comment by mirceaulinic** [comment]
Hey @bharath-ravindranath - it looks like this is a code failure, see https://travis-ci.org/github/napalm-automation/napalm/jobs/677076093#L1044-L1047
To make sure it works, you can execute `tox` locally after installing the dev requirements, e.g., `pip install -r requirements-dev.txt`. Let me know if you have any questions, cheers!

File: napalm/eos/eos.py [hunk 1]
```diff
@@ -805,7 +806,15 @@ def default_neighbor_dict(local_as):
             neighbor_dict.update(
                 {"prefix_limit": {}, "local_as": local_as, "authentication_key": ""}
             )  # few more default values
-            return neighbor_dict
```
Comment (mirceaulinic) [inline napalm/eos/eos.py:808]: I think the failure from https://travis-ci.org/github/napalm-automation/napalm/jobs/677076093#L1044-L1047 is related to this line - this internal function `default_neighbor_dict` must return something, but without this line, it returns None.

Comment (bharath-ravindranath) [reply napalm/eos/eos.py:808]: Thanks @mirceaulinic . That was definitely one of the issues. But it's still failing. Will try and debug this weekend

File: napalm/base/test/models.py [hunk 2]
```diff
@@ -86,6 +86,7 @@
     "remove_private_as": bool,
     "prefix_limit": dict,
     "neighbors": dict,
+    "nhs": bool,
```
Comment (mirceaulinic) [inline napalm/base/test/models.py:89]: This change is going to break the tests for all the other drivers - see, for example, https://travis-ci.org/github/napalm-automation/napalm/jobs/679590322#L501

**Comment by mirceaulinic** [comment]
Hi @bharath-ravindranath - were you going to revisit this? Cheers.

**Pull request closed without merging**